docs/pages/includes/teleport-connect-telemetry.mdx

```diff
@@ -16,6 +16,6 @@ On login, we also collect some device-related data:
 Additionally, we ask for a job role (answer is optional).
 
 The full list of events and collected data is defined as [protocol buffer messages in the Teleport source](https://github.com/gravitational/teleport/tree/branch/v(=teleport.major_version=)/proto/prehog/v1alpha/connect.proto).
-We do not track the details of those events but merely that the given event took place. Each event includes
-the cluster name and user name anonymized with HMAC with the internal cluster ID as the key – it’s impossible for us
-to resolve that anonymized token back to a particular cluster or user.
+We do not track the details of those events but merely that the given event took place. Each event includes the cluster
+name and user name anonymized with HMAC using the cluster's internal random UUID as the key. It is infeasible to
```

Comment:
It might be worth us explaining why we want to track the anonymised cluster and user ID.

Reply:
It allows us to combine the Connect events and Teleport events (although we haven't done it yet), but I'm not sure if this is worth mentioning.

Reply:
It's somewhat my personal opinion that if we want people to opt-in to telemetry, this works better when we provide individual justification for data-points collected - it demonstrates that we are trying to collect the least data possible whilst still implementing useful Telemetry and that we are being responsible about telemetry. For the Machine ID telemetry I explained what data is collected that allows the linking of events, and what data is explicitly not collected: https://goteleport.com/docs/machine-id/reference/telemetry/

Reply:
I'm inclined to leave this as-is for now, but maybe we can consider making a unified docs page for our telemetry data that explains everything in detail? I think I'd rather put the wordsmithing effort into that than an FAQ entry that technically only covers Connect.

Reply:
Hmm having everything in one place sounds nice.

@timothyb89 could you backport this PR to v12? I forgot about it earlier.

Reply:
Yup, I'll backport it